File: vbsrc/Samples/Client_Server/REST_WebClient/REST_ToDoItem.vb
Imports System

Namespace Global.DotNetForHtml5.Showcase.SampleRestWebService.Models
    Public Class ToDoItem
        Public Sub New()
        End Sub

        Public Sub New(ByVal description As String, ByVal ownerId As Integer)
            Me.Description = description
        End Sub

        Public Property Id As Guid
        Public Property OwnerId As Guid
        Public Property Description As String
    End Class
End Namespace

File: src/Samples/Client_Server/REST_HttpClient/REST_ToDoItem.cs
using System;

namespace DotNetForHtml5.Showcase.SampleRestWebService.Models
{
    public class ToDoItem
    {
        public ToDoItem() { }

        public ToDoItem(string description, int ownerId)
        {
            Description = description;
        }

        public Guid Id { get; set; }
        public Guid OwnerId { get; set; }
        public string Description { get; set; }
    }
}